tests/tacacs/utils.py

```diff
@@ -106,6 +110,13 @@ def setup_tacacs_client(duthost, tacacs_creds, tacacs_server_ip):
     return default_tacacs_servers
 
 
+def convert_db_config_to_cli_parameter(cfg):
```

Comment:
>convert_db_config_to_cli_parameter [](http://example.com/codeflow?start=4&length=34)

You may just use `shlex.quote()` instead of implement a new function. And the concept is not related to db_config.

Reply:
Fixed, remove this method and update CONFIG_DB with sonic-db-cli

The shlex.quote can't handle this case. Test framework will backup config_db AAA table before test.  After test, test framework will load AAA table from config_db.json and update running config with 'config aaa' command.
However there is a mismatch between CONFIG_DB data and 'config aaa' command parameter, for example for authorization, the config DB is :
    "authorization": {
        "login": "tacacs+,local"
    }

And the config command is:
config aaa authorization login "local tacacs+"